ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,16 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+Listen 5000
+<VirtualHost *:5000>
+    WSGIScriptAlias / /usr/share/crane/crane_el6.wsgi
+    <Location /crane>
+        Order deny,allow
+        Deny from all
+        Allow from localhost
```

Comment:
That's interesting, I make contact with a remote rhel6 machine configured as-is with a docker client on my machine.

Maybe the docker client itself only ends up trying to get data in `/usr/share/crane` ?

If this change is important for testing, please go ahead and push it and I can spark the jenkins jobs testing this branch (they build with it and then run pulp smash)